config/webpack.dev.js

```diff
@@ -51,13 +37,20 @@ module.exports = (env, argv) => {
             },
             plugins: [
                 // create an html page for every item in ./site/views
-                ...fs.readdirSync("./site/views").map(
-                    (f) =>
-                        new HtmlWebpackPlugin({
-                            template: "./site/views/" + f,
-                            filename: f,
-                        })
-                ),
+                ...fs.readdirSync("./site/views").map((f) => {
+                    const htmlView = fs.readFileSync(
+                        `./site/views/${f}`,
+                        "utf8"
+                    );
+
+                    return new HtmlWebpackPlugin({
+                        templateContent: new Function(
```

Comment:
The use of `new Function` here is a large code smell to me. I'll pull down the PR here shortly and see if I can't whip up an alternative approach.